config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/Bundle.java

```diff
@@ -202,4 +202,26 @@ public void setFolderTree(FolderTree folderTree) {
     public void setDependencyMap(Map<Dependency, List<Dependency>> dependencyMap) {
         this.dependencyMap = dependencyMap;
     }
+
+    public void verifyExistingFolderPath(String folderPath) {
+        boolean isValidImport = false;
+
+        // True because there isn't a specific directory.
+        if (folderPath.equals("/")) {
+            isValidImport = true;
+        }
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this hard-coded path-delimiter. ([squid:S1075](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS1075))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=131&issues=AWct0dyzKqTTJP1B-7zk&open=AWct0dyzKqTTJP1B-7zk)

[ISSUE_KEY]: AWct0dyzKqTTJP1B-7zk

Reply:
@J-Lou add a warning ignore for this squid because this has to be hard-coded.